pages/tree-view/items/permutations-items.tsx

```diff
@@ -21,27 +21,57 @@ export const textItems: Item[] = [
     content: 'Item 1',
     secondaryContent: 'Description 1',
     actions: 'Actions slot',
-    children: [{ id: '1.1', content: 'Item 1.1', secondaryContent: 'Description 1.1', actions: 'Actions slot' }],
+    children: [
+      {
+        id: '1.1',
+        content: 'Item 1.1',
+        secondaryContent: 'Description 1.1',
+        actions: 'Actions slot',
+        children: [
+          {
+            id: '1.2',
```

Comment:
nit: Having item 1.2 nested under 1.1 is confusing. I it better to name it 1.1.1 instead.